test/functional/apps/discover/embeddable/_saved_search_embeddable.ts

```diff
@@ -95,12 +103,41 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should render duplicate saved search embeddables', async () => {
+      await addSearchEmbeddableToDashboard();
```

Comment:
That was actually the problem we had before because we weren't cleaning up between tests, so the duplicate render test actually had 3 embeddables instead of 2, and my replace panel test was failing because by the time it ran there were 4 embeddables.

It's a bit hard to tell because of how the diff renders, but I added a `beforeEach` step to create a new dashboard between each test to ensure we have a fresh state: https://github.com/elastic/kibana/pull/153822/files#diff-851e2afb1c46b682019d3fde3a4fe9cb4e64675650f16c429cdbc238e1263d4fR41.

Reply:
ah, was looking for `afterEach` ... great work!